lib/danger/ci_source/buddybuild.rb

```diff
@@ -1,35 +1,54 @@
 module Danger
   # ### CI Setup
   #
-  # Buddybuild has an integration for Danger already built-in.
-  # What you need to do is to upload your `Gemfile` and `Dangerfile` to
-  # the server and you should be all set-up. However, if you want to use
-  # different bot for Danger, you can do it with token setup described below.
-  #
```

Comment:
We are currently updating the documentation, so this is a chicken or egg situation. I'll check with the team and get back to you.